.github/workflows/main.yml

```
@@ -5,7 +5,7 @@ on:
     branches:
       - main
   pull_request:
-    branches_ignore: []
+    branches-ignore: []
```

actionlint (1.7.4)

"branches-ignore" section should not be empty

(syntax-check)